.github/ISSUE_TEMPLATE/airflow_bug_report.yml

```diff
@@ -51,7 +51,7 @@ body:
       required: true
   - type: textarea
     attributes:
-      label: What you expected to happen
+      label: What you think should happen instead
```

Comment:
```suggestion
      label: What do you think should happen instead
```

Reply:
I think the syntax is correct without _do_. Or if we add _do_ a question mark should also be appended.

Reply:
Correct

Reply:
Leaving as is then :)